near-contract-standards/src/non_fungible_token/events.rs

```diff
@@ -0,0 +1,149 @@
+use std::fmt;
+
+use near_sdk::serde::{Deserialize, Serialize};
+
+//enum that represents the data type of the EventLogJson.
+//the enum can either be an NftMintLog or an NftTransferLog
+#[derive(Serialize, Deserialize, Debug)]
+/*
+    outputs JSON in the following order: <tag>: <field>, <content>: <field_data>
+    In this case an example would be: "event": "NftMint", "data": <NftMintLog Vector Data>
+*/
+#[serde(tag = "event", content = "data")] 
+//turns all the fields into snake case (NftMint --> nft_mint)
+#[serde(rename_all = "snake_case")] 
+#[serde(crate = "near_sdk::serde")]
+//makes it so that adding new fields isn't a breaking change
```

Comment:
I personally don't think the comments here are that helpful and make it harder to read this block. Can keep if you think they are useful, but this was confusing for me to read